taxi/forms.py

```diff
@@ -0,0 +1,14 @@
+from django import forms
+from .models import Car, Manufacturer
+
+
+class CarForm(forms.ModelForm):
+    class Meta:
+        model = Car
+        fields = "__all__"
+
+
+class ManufacturerForm(forms.ModelForm):
+    class Meta:
+        model = Manufacturer
+        fields = "__all__"

```

Comment:
Similar to the CarForm, using `fields = "__all__"` for ManufacturerForm will include all fields from the Manufacturer model. Make sure this is what you want, and consider specifying fields explicitly to avoid exposing sensitive data.